suite-native/device/src/components/UninitializedDeviceModalAppendix.tsx

```diff
@@ -0,0 +1,88 @@
+import { VStack, HStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon, IconName } from '@suite-common/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+import { TxKeyPath } from '@suite-native/intl';
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+const iconBackgroundStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) => ({
+    backgroundColor: utils.colors[color],
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: utils.spacings.extraLarge,
+    height: utils.spacings.extraLarge,
+    borderWidth: color ? utils.borders.widths.small : 0,
+    borderColor: utils.colors.borderElevation1,
+    borderRadius: 12,
+}));
+
+type ListItemProps = {
+    iconBackgroundColor?: Color;
+    translationKey: TxKeyPath;
+    iconColor?: Color;
+} & (
+    | {
+          iconName: IconName;
+          iconNumber?: never;
+      }
+    | {
+          iconName?: never;
+          iconNumber: number;
+      }
+);
```

Comment:
I believe you could use this. IMO it's a little bit more readable when looking at the props. https://github.com/sindresorhus/type-fest/blob/main/source/require-exactly-one.d.ts

Reply:
That is exactly what I was looking for, but I couldn't remember the name, thanks.

Reply:
44bef1e263469b088aaff93b170d0e6f43dc980c